src/providers/krb5/krb5_child_handler.c

```diff
@@ -156,14 +156,14 @@ static errno_t create_send_buffer(struct krb5child_req *kr,
         return ENOMEM;
     }
 
-    buf->size = 8*sizeof(uint32_t) + strlen(kr->upn);
+    buf->size = 9*sizeof(uint32_t) + strlen(kr->upn);
 
     if (kr->pd->cmd == SSS_PAM_AUTHENTICATE ||
         kr->pd->cmd == SSS_PAM_PREAUTH ||
         kr->pd->cmd == SSS_CMD_RENEW ||
         kr->pd->cmd == SSS_PAM_CHAUTHTOK_PRELIM ||
         kr->pd->cmd == SSS_PAM_CHAUTHTOK) {
-        buf->size += 5*sizeof(uint32_t) + strlen(kr->ccname) + strlen(keytab) +
+        buf->size += 4*sizeof(uint32_t) + strlen(kr->ccname) + strlen(keytab) +
```

Comment:
Explanation is trivial. variable `posix_domain` is always passed to krb5-child.child and not just in few cases.

@fidencio  check changes in the commit  861ab44e8148208425b67c4711bc8fade10fd3ed
src/providers/krb5/krb5_child.c -> unpack_buffer
and src/providers/krb5/krb5_child_handler.c -> create_send_buffer

https://pagure.io/SSSD/sssd/blob/master/f/src/providers/krb5/krb5_child_handler.c#_197